src/cpu/flexcpu/inflight_inst.hh

```diff
@@ -55,15 +55,20 @@ class InflightInst : public ExecContext,
     using VecRegContainer = TheISA::VecRegContainer;
     using VecElem = TheISA::VecElem;
 
+    // Mutually exclusive states through which InflightInsts will transition.
```

Comment:
Lol. These need to be all the same part of speech. I don't have concrete suggestions, though.

Reply:
They're all adjectives?